.pre-commit-config.yaml

```diff
@@ -6,10 +6,6 @@ repos:
     language: system
     entry: ./tools/check_changes.py
     pass_filenames: false
-- repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
I believe this one may be deliberate. The reason is that it's best to run fast checks before the heavy ones and if one repo has both heavy and light checks, it's okay to split them.

Reply:
The rest are probably accidental.